prime-router/metadata/schemas/covid-19.schema

```diff
@@ -284,9 +284,9 @@ elements:
     hhsGuidanceField: Ordering provider address
     hl7Field: ORC-24-1
 
-  - name: ordering_provider_street2
+  - name: ordering_provider_street_2
     type: STREET
-    natFlatFileField: Ordering_provider_street2
+    natFlatFileField: Ordering_provider_street_2
     hhsGuidanceField:
```

Comment:
National FlatFile has the field listed as `Ordering_provider_street2`. I'm looking at a file dated August 8, 2020.

Reply:
Bleh.   I hunted around for that but couldn't find it.   We had a mixture of street2 and street_2; I was trying to be consistent.    Sigh.   I'll change it all back....